cmd/cluster/create.go

```diff
@@ -174,7 +180,7 @@ func CreateCluster(ctx context.Context, opts Options) error {
 
 	exampleObjects := apifixtures.ExampleOptions{
 		Namespace:        opts.Namespace,
-		Name:             opts.Name,
+		Name:             infra.Name,
```

Comment:
just to clarify, Is it a hard requirement for the cluster.Name to be the same than the infra.Name?

Reply:
The cluster `Name` is independent the `InfraID`.  The `Name` and `BaseDomain` are used to determine the DNS private zone.

Reply:
It's not clear to me still why `ExampleOptions.Name` is now being changed to `infra.Name`. Isn't `ExampleOptions.Name` supposed to equal the HostedCluster name (represented here by `opts.Name`)?